src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IBinaryOperatorExpression.cs

```diff
@@ -431,5 +431,195 @@ void M(int a, int b, int c, int d, int e, int f, int g, int h, int i, int j, int
 
             VerifyOperationTreeAndDiagnosticsForTest<InvocationExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [Fact]
+        public void LogicalOrFlow_01()
+        {
+            string source = @"
+class P
+{
+    void M(bool a, bool b)
+/*<bind>*/{
+        GetArray()[0] =  a || b;
+    }/*</bind>*/
+
+    static bool[] GetArray() => null;
+}
+";
+            string expectedGraph = @"
+Block[0] - Entry
```

Comment:
Nit: IOperation validation uses two spaces (except for a couple of bugs), but the new logic uses four.

Reply:
> Nit: IOperation validation uses two spaces (except for a couple of bugs), but the new logic uses four.

This is intentional to have a visual separation between blocks and IOperation nodes

---
In reply to: [162167313](https://github.com/dotnet/roslyn/pull/24263#discussion_r162167313) [](ancestors = 162167313)